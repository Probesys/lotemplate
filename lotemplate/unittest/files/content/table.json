{
  "var1": {
    "type": "table",
    "value": [
      "a",
      "b",
      "c",
      "d",
      "e",
      "f",
      "g"
    ]
  },
    "var2": {
    "type": "table",
    "value": [
      "A",
      "B",
      "C",
      "D",
      "E",
      "F",
      "G"
    ]
  }
}