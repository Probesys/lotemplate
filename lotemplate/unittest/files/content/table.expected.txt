
Column1	Column2	Column3
a	static	A
b	static	B
c	static	C
d	static	D
e	static	E
f	static	F
g	static	G

